components/org.wso2.carbon.identity.application.authenticator.basicauth/src/main/java/org/wso2/carbon/identity/application/authenticator/basicauth/BasicAuthenticator.java

```diff
@@ -930,4 +937,10 @@ private String getTenantDomainFromUserName(AuthenticationContext context, String
         }
         return MultitenantUtils.getTenantDomain(username);
     }
+
+    private boolean isURLContainSensitiveData(HttpServletRequest request) {
+
+        return (StringUtils.contains(request.getQueryString(), USERNAME) &&
```

Comment:
Do we need to check whether this is a GET request?

Reply:
Since the credentials can be sent as query params through the POST as well I didn't add that logic here. If we only need to check for GET request. I'll change this logic